README.md

```diff
@@ -207,6 +227,10 @@ expect(getByTestId(container, 'ok-button')).not.toHaveAttribute(
 
 ### `toHaveClass`
 
+```typescript
+toHaveClass(...classNames: string[])
+```
+
 This allows you to check wether the given element has certain classes within its
```

Comment:
Nitpick: `wether` should be `whether` (it's misspelled in a few spots)

Reply:
Nice catch. Thanks!